python/pyspark/pandas/tests/test_spark_functions.py

```diff
@@ -20,25 +20,12 @@
 from pyspark.pandas.spark import functions as SF
 from pyspark.pandas.utils import spark_column_equals
 from pyspark.sql import functions as F
-from pyspark.sql.types import (
-    ByteType,
-    FloatType,
-    IntegerType,
-    LongType,
-)
 from pyspark.testing.pandasutils import PandasOnSparkTestCase
 
 
 class SparkFunctionsTests(PandasOnSparkTestCase):
-    def test_lit(self):
-        self.assertTrue(spark_column_equals(SF.lit(np.int64(1)), F.lit(1).astype(LongType())))
-        self.assertTrue(spark_column_equals(SF.lit(np.int32(1)), F.lit(1).astype(IntegerType())))
-        self.assertTrue(spark_column_equals(SF.lit(np.int8(1)), F.lit(1).astype(ByteType())))
-        self.assertTrue(spark_column_equals(SF.lit(np.byte(1)), F.lit(1).astype(ByteType())))
-        self.assertTrue(
-            spark_column_equals(SF.lit(np.float32(1)), F.lit(float(1)).astype(FloatType()))
-        )
-        self.assertTrue(spark_column_equals(SF.lit(1), F.lit(1)))
+    def test_repeat(self):
```

Comment:
This change is irrelevant to the PR proposal but is made to keep the `test_spark_functions` file.

We shall complete tests for `pyspark/pandas/spark/functions` here later.